src/Microsoft.DotNet.Watcher.Tools/Program.cs

```diff
@@ -29,22 +29,6 @@ public Program(TextWriter consoleOutput, CancellationToken cancellationToken)
 
             _cancellationToken = cancellationToken;
             _out = consoleOutput;
-
-            _loggerFactory = new LoggerFactory();
-
-            var logVar = Environment.GetEnvironmentVariable("DOTNET_WATCH_LOG_LEVEL");
```

Comment:
I don't know why we used environment variable but removing it could break things (I am mostly concerned about some kind of tooling)